ffi/src/lib.rs

```
@@ -302,5 +370,25 @@ fn manager_config(cache_size: usize, revisions: usize, strategy: u8) -> Revision
 /// * `db` - The database handle to close, previously returned from a call to open_db()
 #[unsafe(no_mangle)]
 pub unsafe extern "C" fn fwd_close_db(db: *mut Db) {
+    if db.is_null() {
+        return;
```

nit: should we return an error "already closed" maybe?

---

This is handled in Go - might be better to just remove altogether

---

SGTM, but we should do it in one place consistently.